__tests__/app.test.js

```diff
@@ -395,4 +396,31 @@ describe('9. GET /api/articles Queries', () => {
             })
         })
     })
-})
\ No newline at end of file
+})
+
+describe('10. DELETE /api/comments/:comment_id', () => {
+    test('status:204, responds with an empty response body', () => {
+      return request(app)
+      .delete('/api/comments/2')
+      .expect(204)
+      .then(({body}) => {
+          expect(body.msg).toBe(undefined);
+      });
+    });
+    test('status:404, responds with an error when no such id', () => {
+        return request(app)
+        .delete('/api/comments/999999')
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe('Not Found');
+        });
+      });
+    test('status 404, should return error when format is invalid', () =>{
+        return request(app)
+        .delete('/api/comments/ilikecheese')
+        .expect(400)
+        .then(({body}) => {
+          expect(body.msg).toBe('Bad Request');
+        });
+      });
+  });
```

Comment:
As an optional test it might be good to think about how you might check if the comment has definitely been deleted from the db